cyndi_setup.sql

```diff
@@ -0,0 +1,32 @@
+--
+-- Intializes a Cyndi-like inventory schema and creates a view copied from a
+-- local inventory DB
+--
+
+CREATE SCHEMA IF NOT EXISTS inventory;
+
+CREATE EXTENSION IF NOT EXISTS dblink;
+
+-- This view should be queried instead
+CREATE OR REPLACE VIEW inventory.hosts AS
+SELECT
+    id,
+    account,
```

Comment:
Xref https://github.com/RedHatInsights/compliance-backend/pull/723#discussion_r564421641
Can it be aliased? `account AS account_number`